README.md

```diff
@@ -89,3 +130,13 @@ filebeat_info{version="7.4.0"} 1
 filebeat_libbeat_output{read="errors"} 0
 filebeat_libbeat_output{write="errors"} 0
 ```
+
+
+To expose metrics on from beats-exporter on non-default port, please see example:
```

Comment:
Is this example really needed for such a basic param?

Reply:
Removed example and re-organized readme.